### ISSUE TITLE
Resizing to regions breaks & fixes randomly

### ISSUE THREAD

**Issue by Gappa** [digitalcreations/MaxTo#582]
This bug is strange and I quite don't see a pattern yet. It works and then a few minutes later it doesn't - with the same window/application.

**To Reproduce**

1. Try to maximize a window in a region.
2. It maximizes on the whole display

**Expected behavior**

Windows maximizes to the defined region.

**System information:**

I think it might be connected to the last cumulative update to Win10 version 1909 & newer versions.

However I'm observing the same problem with these setups:

- MaxTo 2.0.1 (stable), Windows 10 1909 (with all the updates to this day).
- MaxTo 2.1.0-beta2, Windows 10 2004.

**Additional context**

Just found out - when it doesn't work, it logs a line for each failure to maximize:

```
2020-06-02 08:03:54 [CompanionX64@2.1.0.660] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "maxto20200602.log – Poznámkový blok" (0x"d055c"), used mouse position to find region "{X=1920,Y=0,Width=1379,Height=1080}".
2020-06-02 08:04:39 [CompanionX64@2.1.0.660] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "Hangouts Google" (0x"1917d6"), used mouse position to find region "{X=3299,Y=540,Width=541,Height=540}".
2020-06-02 08:04:50 [CompanionX64@2.1.0.660] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "maxto20200602.log – Poznámkový blok" (0x"41874"), used mouse position to find region "{X=1920,Y=0,Width=1379,Height=1080}".
2020-06-02 08:05:10 [CompanionX64@2.1.0.660] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "maxto20200602.log – Poznámkový blok" (0x"9186a"), used mouse position to find region "{X=1920,Y=0,Width=1379,Height=1080}".
```

(This log is from the machine with Windows 10 2004 & 2.10-beta2)

` Poznámkový blok` = Notepad.exe

**Comment by Gappa** [comment]
Same problem on the other machine/setup:

```
2020-06-02 18:02:19 [CompanionX64@2.1.0.660] [Information] [MaxTo.Companion.MessageHandlers.DragWindowMessageHandler] Did not receive any moving events while dragging "Google Chrome" (0x"30716"), used mouse position to find region "{X=0,Y=0,Width=2573,Height=2100}".
``

**Comment by gorbunovav** [comment]
I can confirm this bug. And I see the same lines in the log.
My version of the app is `app-2.1.0-beta2` too.

Maybe it is related to the recent app update (or some recent minor Windows updates).
Because I am on the Slow Insider ring for a long time and it was working ok until recently.

Temporal fix, which works for me: open the "Start" menu (I am using the "Win" keyboard key if it matters).

I will try to rollback to `app-2.1.0-beta1` and check how it behaves.

**Comment by Gappa** [comment]
> Temporal fix, which works for me: open the "Start" menu (I am using the "Win" keyboard key if it matters).

Wow, it really works, just tested it :)

**Comment by vegardlarsen** [comment]
I have no idea what causes this, but I will investigate. Thank you for the Start menu tip, that could prove to be very significant.

**Comment by gorbunovav** [comment]
@vegardlarsen I can confirm, that it doesn't happen for me with `app-2.1.0-beta1`

**Comment by vegardlarsen** [comment]
I believe I have fixed this for 2.1.0-beta.4.

**Comment by gorbunovav** [comment]
@vegardlarsen works for me. Thank you!
